conformance/apis/v1beta1/profilereport.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 // ProfileReport is the generated report for the test results of a specific
 // named conformance profile.
```

Comment:
Any harm in moving away from the "Conformance Profiles" term and just using "Conformance Report" here? If I'm remembering correctly, we've basically just said that you support a resource or not, but have mostly moved away from profiles that group resources together?

Reply:
I'm not sure I'm getting your point here, but `ConformanceProfiles` in the report is still a slice containing multiple profile results: 

https://github.com/kubernetes-sigs/gateway-api/blob/7c672a75aaf5e66b05f9df5f2c7b3ce2d8e03d9f/conformance/apis/v1alpha1/conformancereport.go#L44-L46